### PR TITLE
Proper Docker image with 17.1 test db

### PR DESCRIPTION
**Pull request by PiotrZakrzewski**

### CODE REVIEW

**Comment by wardweistra** [comment]
@PiotrZakrzewski I get:
```
Wards-MacBook-Pro-3:docker ward$ docker --version
Docker version 1.12.3, build 6b644ec
Wards-MacBook-Pro-3:docker ward$ docker-compose up
ERROR: In file './docker-compose.yml' service 'version' doesn't have any configuration options. All top level keys in your docker-compose.yml must map to a dictionary of configuration options.
```

Both with docker or solr branch.

**Comment by PiotrZakrzewski** [comment]
Can you check your docker-compose version?
It works for me and Jarno, we installed docker recently
 
    docker-compose version 1.8.1, build 878cff1

File: docker/tm_test_db/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+RUN git clone -b $BRANCH --depth 1 https://github.com/thehyve/transmart-upgrade.git /tmp/transmart-upgrade
+
+ENV POSTGRES_PASSWORD ''
+
+RUN ln -snf /root/.sdkman/candidates/groovy/current/bin/groovy /bin/groovy
+RUN echo "cd /tmp/transmart-upgrade/transmart-data && \
+ make -C env ../vars && source /tmp/transmart-upgrade/transmart-data/vars && \
+ make postgres_drop || true && \
+ make postgres_test || true && \
```
Comment (forus) [inline docker/tm_test_db/Dockerfile]: @PiotrZakrzewski Have you seen the following message? https://github.com/thehyve/transmart-upgrade/blob/master/transmart-data/vars.sample#L20

**Comment by gijskant** [comment]
Same for me with `Docker version 1.12.3, build 6b644ec`

**Pull request merged**